Kudu.Contracts/Scan/ScanStatusResult.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Kudu.Contracts.Infrastructure;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Kudu.Contracts.Scan
+{
+    public class ScanStatusResult : INamedObject
```

Comment:
Do we need to inherit `INamedObject` ?

Reply:
Same as https://github.com/Azure-App-Service/KuduLite/pull/55#discussion_r297361558.